Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1303,6 +1305,36 @@ inline GridAutoFlow BuilderConverter::convertGridAutoFlow(BuilderState&, const C
     return autoFlow;
 }
 
+inline Vector<StyleContentAlignmentData> BuilderConverter::convertAlignTracks(BuilderState& builder, const CSSValue& value)
+{
+    auto& valueList = downcast<CSSValueList>(value);
+
+    Vector<StyleContentAlignmentData> alignTracks;
+    alignTracks.reserveInitialCapacity(valueList.length());
+
+    for (auto val : valueList) {
+        auto& contentDistributionValue = downcast<CSSContentDistributionValue>(val.get());
+        alignTracks.append(convertContentAlignmentData(builder, contentDistributionValue));
+    }
+
+    return alignTracks;
+}
+
+inline Vector<StyleContentAlignmentData> BuilderConverter::convertJustifyTracks(BuilderState& builder, const CSSValue& value)
```

Comment:
This function is identical to the previous one, can we rename the previous one to `convertContentAlignmentDataList` and set the converter in CSSProperties.json respectively?